CoinDock-api/app/Models/V1/Wallet.php

```diff
@@ -4,23 +4,18 @@
 
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+use App\Http\Controllers\V1\CoinsController;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+use phpDocumentor\Reflection\Types\Null_;
 
 class Wallet extends Model
 {
     use HasFactory;
 
-    protected $fillable = [
-        'coin_id',
-        'user_id',
-        'wallet_id',
-        'balance'
-    ];
-
```

Comment:
why was the $fillable variable removed? Any reason behind it.

Reply:
no reasons ,modified